portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase

```diff
@@ -1319,6 +1319,84 @@ definition {
 		Alert.viewErrorMessage(errorMessage = "Your request failed to complete.");
 	}
 
+	@description = "Assert that items are being properly indexed if they exists both in Production and in a Publication"
+	@priority = 3
+	test ViewItemsAreProperlyIndexedInProductionAndPublication {
+		task ("Create a new tag") {
+			JSONAssettag.addTag(
+				groupName = "Guest",
+				tagName = "tag1");
+		}
+
+		task ("Create a Web Content with a tag") {
+			JSONWebcontent.addWebContent(
+				assetTagNames = "tag1",
+				content = "WC WebContent Content",
+				groupName = "Guest",
+				title = "WC WebContent Title");
+		}
+
+		task ("Add a Widget Page with Asset Publisher") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Page",
+				widgetName = "Asset Publisher");
+		}
+
+		task ("Select the Web Content Article as Asset Type and assert that 'WC WebContent Title' is being returned by the Asset Publisher") {
+			Refresh();
+
+			Navigator.gotoPage(pageName = "Test Page");
+
+			AssetPublisherPortlet.configureAssetTypePG(
+				assetSubtype = "Any",
+				selectedAsset = "Web Content Article");
+
+			Button.clickSave();
+
+			Button.clickCancel();
+
+			AssetPublisherPortlet.viewAssetPG(assetTitle = "WC WebContent Title");
+		}
+
+		task ("Add new Publication, edit the tag inside the publication and change back to production") {
+			JSONPublications.addPublication(publicationName = "Publication Name");
+
+			JSONPublications.selectPublication(publicationName = "Publication Name");
+
+			Tag.openTagsAdmin(siteURLKey = "guest");
+
+			Tag.viewCP(tagNameList = "tag1");
```

Comment:
This assertion isn't needed because `Tag.editCP` will fail if the tag doesn't exist